projects/core/src/organization/store/reducers/org-unit.reducer.ts

```diff
@@ -19,8 +19,6 @@ export function orgUnitListReducer(
   action: LoaderAction
 ): any {
   switch (action.type) {
```

Comment:
Why are empty switch statements here in this file?

Reply:
most of that reducers are not used, but could be if we will not get updated api, lets do cleanup at the end of epic